### PR TITLE
OHARA-1378 Add a select in FTP source connector for user to select co…

### PR DESCRIPTION
**Pull request by DawnJheng**
…nnections.

### CODE REVIEW

**Comment by DawnJheng** [comment]
run

**Comment by konekoya** [comment]
Sorry @DawnJheng, I should have told you that I've re-wrote some of the logic of HDFS sink connector. Could you make some tweaks to this PR so it can have the same logic as HDFS sink connector?

**Comment by DawnJheng** [comment]
Please review again, thanks.

**Comment by konekoya** [comment]
Will have a look later. Thanks for updating it!

**Comment by konekoya** [comment]
@DawnJheng I'm asking @eechih to help review this PR. Let's get this PR going :)

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/FtpSource.js [hunk 1]
```diff
@@ -243,6 +248,28 @@ class FtpSource extends React.Component {
     }
   };
 
+  fetchFtp = async () => {
+    const { currConnection } = this.state;
+    const res = await configurationApis.fetchFtp();
+    const connections = await _.get(res, 'data.result', []);
```
Comment (eechih) [inline ohara-manager/client/src/components/pages/PipelinePage/connectors/FtpSource.js]: _.get() 是同步的 func，不用加 await

**Comment by eechih** [comment]
@DawnJheng Please help rebase onto master.

**Comment by eechih** [comment]
We are going to close it and open an issue to reference it.

https://is-land.atlassian.net/browse/OHARA-1461

**Pull request closed without merging**